python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/graphql_helpers.py

```diff
@@ -134,3 +134,17 @@ def parse_columns(entity: dict[str, Any]) -> list[dict[str, Any]]:
 
     # Sort primary keys first, then sort alphabetically
     return sorted(result, key=lambda c: (0 if c["isPrimaryKey"] else 1, c["name"]))
+
+
+def parse_relations(relations_dict) -> Tuple[int, list[dict]]:
```

Comment:
I don't think this type is correct as it's a dict now.

But it might be simpler to just return the list here and ignore the total. I think the total would only be used if we need to page through a large set of relations?

Reply:
Yeah have fixed, maybe we could keep it in as if we use relations going the other direction we'd want it and not doing too much harm in there

Reply:
or maybe it is complexity we don't need at the moment. Front end has no use for it so it's kind of redundant at the moment